bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/SunSpecConfiguration.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.sunspec.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link SunSpecConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Nagy Attila Gábor - Initial contribution
+ */
+@NonNullByDefault
+public class SunSpecConfiguration {
+
+    /**
+     * Refresh interval in seconds
+     */
+    private long refresh;
```

Comment:
- add the defaults here
- usually our onfiguration classes have no setters and getters but public fields. is there a special reason why you chose do use another pattern?

Reply:
ok, added a default.

There is no special reason to use getters/setters, so removed them.